packages/abcde.rb

```diff
@@ -21,7 +21,7 @@ class Abcde < Package
   })
 
   def self.build
-    system "sed -i 's,prefix = /usr/local,prefix = #{CREW_DEST_PREFIX},' Makefile"
+    system "sed -i 's,prefix = #{CREW_PREFIX},prefix = #{CREW_DEST_PREFIX},' Makefile"
```

Comment:
This is already correct.  What this is doing is abstracting the hardcoded path in Makefile.  Your change will break this if `CREW_PREFIX` is anything other than `/usr/local`.